common/test/acceptance/pages/studio/library.py

```diff
@@ -168,6 +169,7 @@ def count(self, count):
         Sets value of children count input
         """
         count_text = self.get_setting_element(self.COUNT_LABEL)
+        count_text.send_keys(Keys.BACK_SPACE)
```

Comment:
Okay sure I will address your comments and will change it to select all then delete (and remove the .clear() method)